package.json

```diff
@@ -1217,7 +1217,8 @@
         "compile:view": "webpack --mode none --config src/components/logs/webpack.config.js",
         "watch": "webpack --mode development --watch --info-verbosity verbose",
         "test-compile": "tsc -p ./",
-        "test": "npm run test-compile && node ./out/test/runTest.js"
+        "test": "npm run test-compile && node ./out/test/runTest.js",
+        "preinstall": "npx npm-force-resolutions"
```

Comment:
which dependency requires this?

Reply:
npx is used to force every dependency in resolutions:
"resolutions": {
        "minimist": "1.2.5",
        "yargs-parser": "13.1.2",
        "ini": "1.3.8",
        "glob-parent": "5.1.2",
        "normalize-url": "4.5.1"
}
offending versions are included through following dependencies:
* minimist: gulp -> glob-watchers -> chokidar -> fsevents -> node-pre-gyp -> (mkdirp, rc) -> minimist
*  yargs-parser: gulp -> gulp-cli -> yargs -> yargs-parser
* ini: gulp -> glob-watchers -> chokidar -> fsevents -> node-pre-gyp -> rc -> ini
* glob-parent: gulp > vinyl-fs > glob-stream > glob-parent, gulp > glob-watcher > chokidar > glob-parent
* normalize-url: download -> got -> cacheable-request -> normalize-url